source/gui/settingsDialogs.py

```diff
@@ -2815,7 +2815,7 @@ def haveConfigDefaultsBeenRestored(self):
 			and self.ConsoleUIACheckBox.IsChecked() == (self.ConsoleUIACheckBox.defaultValue == 'UIA')
 			and self.winConsoleSpeakPasswordsCheckBox.IsChecked() == self.winConsoleSpeakPasswordsCheckBox.defaultValue
 			and self.cancelExpiredFocusSpeechCombo.GetSelection() == self.cancelExpiredFocusSpeechCombo.defaultValue
-			and self.UIAInChromiumCombo.selection == self.UIAInChromiumCombo.defaultValue
+			and self.UIAInChromiumCombo.GetSelection == self.UIAInChromiumCombo.defaultValue

```

Comment:
```suggestion
			and self.UIAInChromiumCombo.GetSelection() == self.UIAInChromiumCombo.defaultValue

```

It should be this, right?